editor.planx.uk/src/@planx/components/Send/Editor.tsx

```diff
@@ -45,28 +50,11 @@ const SendComponent: React.FC<Props> = (props) => {
     }),
   });
 
-  const options: { value: SendIntegration; label: string }[] = [
-    {
-      value: "bops",
-      label: "BOPS",
-    },
-    {
-      value: "uniform",
-      label: "Uniform",
-    },
-    {
-      value: "idox",
-      label: "Idox Nexus (TESTING ONLY)",
-    },
-    {
-      value: "email",
-      label: "Email to planning office",
-    },
-    {
-      value: "s3",
-      label: "Upload to AWS S3 bucket",
-    },
-  ];
```

Comment:
Was a bit of a pain to manage variations of rich text and string-templated descriptions & labels as a list like this - so, similar to `SetFee`, am just hardcoding each option in the `return` rather than using `map`